src/sources.js

```diff
@@ -315,6 +315,11 @@ class S3Source extends Source {
         .split("_")
         .join("/"));
   }
+  async availableLogo() {
+    const objects = await this._listObjects();
```

Comment:
In future PRs we'll want to (try to) get a customised title, byline etc which will also be stored in the bucket and therefore will also use the output of `this._listObjects()`. Very happy to defer that to future PRs if you'd prefer?

Reply:
I moved it to `getInfo()` in the new commit